Autopilot/AttitudeManager/fetchSensorMeasurementsMode.hpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * Fetch Sensor Measurements Mode: gets raw sensor data from the sensor drivers
```

Comment:
I think we don't need an entire module dedicated to doing this. All we're really doing is calling the getData functions from the sensors. It doesen't really hurt anyone if we keep this stuff in a module like this, but yeah, sort of overkill and I'd start getting worried about clutter.

Reply:
Only reason I made a separate module is so that any calls to the sensor drivers are separated from the rest of the rest of the attitude manager. 

The point you bring up about clutter is fair, though. One solution I can think of is just putting the code in the `SensorMeasurements_GetResult()` function into the `fetchSensorMeasurementsMode::execute()` method. However, if I do this then I might as well ditch the new structs I made since I'll need to `#include "IMU.hpp"` and all in `attitudeStateClasses.hpp`